scripts/core/editor3/highlightsConfig.js

```diff
@@ -12,18 +12,51 @@ export const highlightsConfig = {
         }
     },
     ADD_SUGGESTION: {
-        type: 'ADD_SUGGESTION',
+        type: 'CHANGE',
+        description: 'Add',
         draftStyleMap: {
             color: 'rgba(0, 180, 0, 1.0)'
         }
     },
     DELETE_SUGGESTION: {
-        type: 'DELETE_SUGGESTION',
+        type: 'CHANGE',
+        description: 'Remove',
         draftStyleMap: {
             color: 'rgba(255, 0, 0, 1.0)',
             textDecoration: 'line-through'
         }
     },
+    TOGGLE_BOLD_SUGGESTION: {
```

Comment:
TOGGLE_BOLD_SUGGESTION is a new suggestion of type 'STYLE'.
All new STYLE suggestions are defined only once on this file.
STYLE type of suggestions was needed in order to support suggestion on the same character one of change (ADD/DELETE SUGGESTIONS) and one or more STYLE suggestions.
I don't see any change that should be done here.

Reply:
Yeah, it might be too much work to make style suggestions to work with any styles out of the box, and definitions in a single file make it less an issue.